docs/personas.md

```diff
@@ -0,0 +1,25 @@
+# Personas
+## [P1] <a name="P1">Propietario</a>
+Dueño y CEO del restaurante. Como propietario es el encargado de tomar decisiones acertadas para el futuro de su negocio, además de gestionar el personal. 
+
+## [P2] <a name="P2">Camarero</a>
+Un empleado del restaurante. Encargado de atender de forma a las mesas, servir la comida hecha por el [cocinero](#P3) y del trato con los [clientes](#P4).
+
+## [P3] <a name="P3">Cocinero</a>
+Empleado del restaurante encargado de hacer las comidas que piden los clientes.
```

Comment:
Se os pide que modelicéis el problema, y aquí no veo más que una aproximación relativamente simple al mismo. Para empezar, la modelización de los diferentes tipos de usuarios es para poder ofrecer una lógica de negocio y un beneficio. El beneficio que los empleados del restaurante podrían obtener (si concretaras exactamente cuál es la lógica de negocio) será siempre el mismo.